docs/source/usersguide/tallies.rst

```diff
@@ -0,0 +1,311 @@
+    dE}_{\text{filters}} \underbrace{f(\mathbf{r}, \mathbf{\Omega},
+    E)}_{\text{scores}} \psi (\mathbf{r}, \mathbf{\Omega}, E)
+
+Thus, to specify a tally, we need to specify what regions of phase space should
+be included when deciding whether to score an event as well as what the scoring
+function (:math:`f` in the above equation) should be used. The regions of phase
+space are called *filters* and the scoring functions are simply called *scores*.
+
+-------
+Filters
+-------
+
+To specify the regions of phase space, one must create a
+:class:`openmc.Filter`. Since :class:`openmc.Filter` is an abstract class, you
+actually need to instantiate one of its sub-classes (for a full listing, see
+:ref:`pythonapi_tallies`). For example, to indicate that events that occur in a
+given cell should score to the tally, we would create a
+:class:`openmc.CellFilter`::
+
+   cell_filter = openmc.CellFilter([fuel.id, moderator.id, reflector.id])
+
+Another commonly used filter is :class:`openmc.EnergyFilter`, which specifies
+multiple energy bins over which events should be scored. Thus, if we wanted to
+tally events where the incident particle has an energy in the ranges [0 eV, 4
+eV] and [4 eV, 1 MeV], we would do the following::
+
+  energy_filter = openmc.EnergyFilter([0.0, 4.0, 1.0e6])
+
+Energies are specified in eV and need to be monotonically increasing.
+
+.. caution:: An energy bin between zero and the lowest energy specified is not
+             included by default as it is in MCNP.
+
+Once you have created a filter, it should be assigned to a :class:`openmc.Tally`
+instance through the :attr:`Tally.filters` attribute::
+
+  tally.filters.append(cell_filter)
+  tally.filters.append(energy_filter)
+
+  # ..or..
+  tally.filters = [cell_filter, energy_filter]
+
+.. note:: You are actually not required to assign any filters to a tally. If you
+          create a tally with no filters, all events will score to the
+          tally. This can be useful if you want to know, for example, a reaction
+          rate over your entire model.
+
+.. _usersguide_scores:
+
+------
+Scores
+------
+
+To specify the scoring functions, a list of strings needs to be given to the
+:attr:`Tally.scores` attribute. You can score the flux ('flux'), a reaction rate
+('total', 'fission', etc.), or even scattering moments (e.g., 'scatter-P3'). For
+example, to tally the elastic scattering rate and the fission neutron
+production, you'd assign::
+
+  tally.scores = ['elastic', 'nu-fission']
+
+With no further specification, you will get the total elastic scattering rate
+and the total fission neutron production. If you want reaction rates for a
+particular nuclide or set of nuclides, you can set the :attr:`Tally.nuclides`
+attribute to a list of strings indicating which nuclides. The nuclide names
+should follow the same :ref:`naming convention <usersguide_naming>` as that used
+for material specification. If we wanted the reaction rates only for U235 and
```

Comment:
Does this mean that the reaction rates for U235 and U238 are combined together into a single tally, or are tallied separately? Of course we both know that the latter is the case, but the wording is ambiguous for a new user.